src/main/FizzBuzz.java

```diff
@@ -0,0 +1,24 @@
+package main;
+
+public class FizzBuzz {
+    public static void main(String[] args) {
+        for (int i=1; i<=100; i++){
+            if (i%3==0 && i%5==0){
+                System.out.println("FizBuzz");
+            }
+            if (i%3==0){
+                System.out.println("Fizz");
+            }
+            if (i%5==0){
+                System.out.println("Buzz");
+            }
+            if (i==15){
```

Comment:
There is no need to use this condition, because first condition handles all numbers which can be devided by 3 or 5.